python/wiki_etl.py

```diff
@@ -0,0 +1,417 @@
+"""
+Usage:
+    wiki_etl [options] <wikidb> <day>
+
+Download an XML dump (if available), converts it to json (sorting revisions
+by timestamp for each page), then create and load associated fulltext and
+metadata hive tables (default values are set for enwiki size) .
+
+xml dump path:     <base-path>/<wikidb>-<day>/xmlbz2
+sorted json path:  <base-path>/<wikidb>-<day>/jsonbz2
+metadata path:     <base-path>/<wikidb>-<day>/meta_parquet
+
+hive fulltext table: <db>.<wikidb>_<day>_fulltext
+hive metadata table: <db>.<wikidb>_<day>
+
+Arguments:
+    <wikidb>                 The wiki to process, wikidb format, like enwiki
+    <day>                    The day to check dump for, yyyyMMdd format
+
+Options:
+    --no-download            If set, doesn't try to download dump, go straight
+                              to xmljson extraction
+    --no-xmljson             If  set, doesn't try to extract json from xml, go
+                              straight to hive tables creation
+    --no-fulltext-table      If set, doesn't create hive fulltext table
+    --no-metadata-table      If set, doesn't create hive metadata table
+    --no-metadata-load       If set, doesn't laod hive metadata table
+                               from fulltext one
```

Comment:
Are we ever going to want `--no-xmljson` but not also want `--no-download`? Would it work to have something like a `--skipto=<stage>` and `--stop=<stage>` rather than needing to specify all of the steps that we don't want to do?